packages/medusa-react/src/hooks/admin/payments/queries.ts

```diff
@@ -0,0 +1,31 @@
+import { queryKeysFactory } from "../../utils"
+import { AdminPaymentRes } from "@medusajs/medusa"
+import { useQuery } from "react-query"
+import { useMedusa } from "../../../contexts"
+import { UseQueryOptionsWrapper } from "../../../types"
+import { Response } from "@medusajs/medusa-js"
+
+const PAYMENT_QUERY_KEY = `payment` as const
+
+export const paymentQueryKeys =
+  queryKeysFactory<typeof PAYMENT_QUERY_KEY>(PAYMENT_QUERY_KEY)
+
+type PaymentKey = typeof paymentQueryKeys
+
+export const usePayment = (
```

Comment:
todo: Missing domain from name
```suggestion
export const useAdminPayment = (
```